cra-client/.env.sample

```diff
@@ -7,6 +7,7 @@
 # requires use of HTTPS protocol
 HTTPS=true
 # location of cert file in codebase, required to properly use HTTPS locally
-SSL_CRT_FILE=../ssl/keys/server.cert
+
+SSL_CRT_FILE=../../ssl/keys/EgeriaReactUIClient.cert.pem
 # location of key file in codebase, required to properly use HTTPS locally
-SSL_KEY_FILE=../ssl/keys/server.key
\ No newline at end of file
+SSL_KEY_FILE=../../ssl/keys/EgeriaReactUIClient.key.pem
```

Comment:
is this code running in the user's browser?
If so, whilst we *could* use a client certificate, often this isn't used (as the client needs to check the server is appropriate, but not the other way around).

(I'm trying to put the architecture in the context of the ssl sequence diagrams)

And if used, this would normally be managed by the user in the browser?

And the same would apply for the CA we use to validate the connection to any server -- in this case I presume. This would only need configuring if the environment is using a self-signed cert and that hasn't been 'installed' on the system/browser via a device management type process. But it doesn't make sense to provide this content to the client app, as that defeats the purpose of using certificates?

My comments may have/are confusing as I'm not familiar enough with the create-react-app architecture.